examples/iscsi/README.md

```diff
@@ -1,15 +1,16 @@
-# How to Use it?
-Here is my setup to setup Kubernetes with iSCSI persistent storage. I use Fedora 21 on Kubernetes node. 
-
-Install iSCSI initiator on the node:
+## Step 1. Setting up iSCSI target and iSCSI initiator
```

Comment:
How about splitting this into Step 1.a Setup on Fedora and Step 1.b Set up on GCE?